azure-pipelines.yml

```diff
@@ -20,7 +20,7 @@ stages:
   jobs:
   - job:
     pool: sonictest
-    displayName: "kvmtest-t0"
+    displayName: "kvmtest-t0-part1"
     timeoutInMinutes: 400
```

Comment:
Probably we can consider reducing this timeout value now.

Reply:
Good point, update it to 200.